borsh/src/de/mod.rs

```diff
@@ -465,17 +469,6 @@ where
     }
 }
 
-impl<T> BorshDeserialize for BinaryHeap<T>
-where
-    T: BorshDeserialize + Ord,
-{
-    #[inline]
-    fn deserialize_reader<R: Read>(reader: &mut R) -> Result<Self> {
-        let vec = <Vec<T>>::deserialize_reader(reader)?;
-        Ok(vec.into_iter().collect::<BinaryHeap<T>>())
-    }
-}
-
```

Comment:
Could you, please, create a separate PR for dropping BinaryHeap support, so it is easier to discover it if anyone would need to read about this change?